ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -366,4 +378,8 @@ extension DrawerDemoController: DrawerControllerDelegate {
         // reset the flag once drawer gets dismissed
         shouldConfirmDrawerDismissal = false
     }
+
+    func drawerControllerDidChangeExpandedState(_ controller: DrawerController) {
+        expandButton?.setTitle(controller.isExpanded ? "Return to normal" : "Expand", for: .normal)
```

Comment:
Should we separate the strings out to constants? Just for good style in our test app.